lifecycle_msgs/msg/State.msg

```diff
@@ -1,26 +1,57 @@
-# Primary state definitions
-# as depicted in
+# Primary state definitions as depicted in
 # http://design.ros2.org/articles/node_lifecycle.html
 
-# These are the primary states
-# State changes can only be requested
-# when the node is in one of these states.
+# These are the primary states. State changes can only be requested when the
+# node is in one of these states.
+
+# Indicates state has not yet been set
 uint8 PRIMARY_STATE_UNKNOWN = 0
+
+# This is the life cycle state the node is in immediately after being
```

Comment:
```suggestion
# This is the lifecycle state the node is in immediately after being
```

Reply:
Life cycle is two words, not sure why the package name uses one